testpar/testphdf5.h

```diff
@@ -195,78 +195,76 @@ extern int facc_type;            /*Test file access type */
 
 /* Test program prototypes */
-void test_plist_ed(const void *params);
-void external_links(const void *params);
-void zero_dim_dset(const void *params);
-void test_file_properties(const void *params);
-void test_delete(const void *params);
-void test_invalid_libver_bounds_file_close_assert(const void *params);
-void test_evict_on_close_parallel_unsupp(const void *params);
-void test_fapl_preserve_hints(const void *params);
-void multiple_dset_write(const void *params);
-void multiple_group_write(const void *params);
-void multiple_group_read(const void *params);
-void collective_group_write_independent_group_read(const void *params);
-void collective_group_write(const void *params);
-void independent_group_read(const void *params);
-void test_fapl_mpio_dup(const void *params);
-void test_get_dxpl_mpio(const void *params);
-void test_split_comm_access(const void *params);
-void test_page_buffer_access(const void *params);
-void dataset_atomicity(const void *params);
-void dataset_writeInd(const void *params);
-void dataset_writeAll(const void *params);
-void extend_writeInd(const void *params);
-void extend_writeInd2(const void *params);
-void extend_writeAll(const void *params);
-void dataset_readInd(const void *params);
-void dataset_readAll(const void *params);
-void extend_readInd(const void *params);
-void extend_readAll(const void *params);
-void none_selection_chunk(const void *params);
-void actual_io_mode_tests(const void *params);
-void no_collective_cause_tests(const void *params);
-void test_chunk_alloc(const void *params);
-void test_chunk_alloc_incr_ser_to_par(const void *params);
-void test_filter_read(const void *params);
-void compact_dataset(const void *params);
-void null_dataset(const void *params);
-void big_dataset(const void *params);
-void dataset_fillvalue(const void *params);
-void coll_chunk1(const void *params);
-void coll_chunk2(const void *params);
-void coll_chunk3(const void *params);
-void coll_chunk4(const void *params);
-void coll_chunk5(const void *params);
-void coll_chunk6(const void *params);
-void coll_chunk7(const void *params);
-void coll_chunk8(const void *params);
-void coll_chunk9(const void *params);
-void coll_chunk10(const void *params);
-void coll_irregular_cont_read(const void *params);
-void coll_irregular_cont_write(const void *params);
-void coll_irregular_simple_chunk_read(const void *params);
-void coll_irregular_simple_chunk_write(const void *params);
-void coll_irregular_complex_chunk_read(const void *params);
-void coll_irregular_complex_chunk_write(const void *params);
-void io_mode_confusion(const void *params);
-void rr_obj_hdr_flush_confusion(const void *params);
-void chunk_align_bug_1(const void *params);
-void lower_dim_size_comp_test(const void *params);
-void link_chunk_collective_io_test(const void *params);
-void contig_hyperslab_dr_pio_test(ShapeSameTestMethods sstest_type);
-void checker_board_hyperslab_dr_pio_test(ShapeSameTestMethods sstest_type);
```

Comment:
Removed the 2 prototypes above, as they are static (and named differently) in `testpar/t_shapesame.c` and useless here.